src/jtagmkI.c

```diff
@@ -1158,7 +1189,8 @@ static void jtagmkI_display(const PROGRAMMER *pgm, const char *p) {
 
```

Comment:
Just a quick thought: Let the computer do this nasty work. Could a format like this help to maintain an uniform appearance more easily instead of manual space counting? 
```c
const int msg_space = -22;
msg_info("%s%*s: 0x%02x\n", p, msg_space, "ICE HW version", hw);
```